src/main/java/io/federecio/dropwizard/sample/SampleResource.java

```diff
@@ -1,23 +1,32 @@
 package io.federecio.dropwizard.sample;
 
 import javax.ws.rs.GET;
+import javax.ws.rs.HeaderParam;
 import javax.ws.rs.Path;
 import javax.ws.rs.Produces;
 import javax.ws.rs.core.MediaType;
 import io.dropwizard.auth.Auth;
 import io.dropwizard.auth.PrincipalImpl;
 import io.swagger.annotations.Api;
 import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
 import io.swagger.annotations.ApiResponse;
 import io.swagger.annotations.ApiResponses;
+import io.swagger.annotations.Authorization;
 import io.swagger.annotations.BasicAuthDefinition;
+import io.swagger.annotations.OAuth2Definition;
 import io.swagger.annotations.SecurityDefinition;
 import io.swagger.annotations.SwaggerDefinition;
 
 @Api("/")
 @Path("/")
 @SwaggerDefinition(securityDefinition = @SecurityDefinition(basicAuthDefinions = {
-        @BasicAuthDefinition(key = "sample") }))
+        @BasicAuthDefinition(key = "basic") }, oAuth2Definitions = {
+    @OAuth2Definition(
+        flow = OAuth2Definition.Flow.IMPLICIT,
+        key = "oauth2",
+        authorizationUrl = "http://brentertainment.com/oauth2/lockdin/authorize",
```

Comment:
can these URLs be removed in favor of tying this into the sample application somehow or would that require more work?